content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -32,6 +32,12 @@ in that there is no corresponding InfluxDB OSS release.
 - Ensure a newline is printed after a successful copy during [restoration](/enterprise_influxdb/v1.9/administration/backup-and-restore/).
 - Make `entropy show` expiration times consistent with `show-shards`.
 
+### Other changes
+
+- Remove `influx_stress` tool. (This had been marked as deprecated since version 1.2.)
+  Instead, use [`inch`](https://github.com/influxdata/inch) 
+  or [`influx-stress`](https://github.com/influxdata/influx-stress) (not to be confused with `influx_stress`).
```

Comment:
Oh man, do we have 2 tools with the same name that do different things? (searched docs for underscore version "influx_stress" and didn't find anything...)

Reply:
From what I've been told, yes, we've two things with similar names (`-` vs `_`).

Reply:
Yikes. Thanks, Will....let's ask whoever told you yes:
- Is it possible to rename and/or do we have an issue for this? (Seems like undesirable UX)
- What is the distinction between the two, so we can fully document? (A search on influx_stress in docs returns crickets)